Microsoft.Toolkit.Uwp.UI.Controls/PullToRefreshListView/PullToRefreshListView.cs

```diff
@@ -142,6 +154,11 @@ protected override void OnApplyTemplate()
                 _refreshIndicatorTransform != null &&
                 _defaultIndicatorContent != null)
             {
+                _root.ManipulationMode = Windows.UI.Xaml.Input.ManipulationModes.TranslateY;
+                _root.ManipulationDelta += Scroller_ManipulationDelta;
+                _root.ManipulationStarted += Scroller_ManipulationStarted;
+                _root.ManipulationCompleted += Scroller_ManipulationCompleted;
```

Comment:
This seems like a hack around DirectManipulation. Wouldn't it be better to use [ProcessInput](https://msdn.microsoft.com/en-us/library/windows/desktop/hh446969(v=vs.85).aspx#processing_keyboard_and_mouse_input) to forward the mouse events to direct manipulation so that the rest of the code just stays the same?